File: app/src/main/java/com/koczuba/popularmovies/data/MovieListResponse.java
package com.koczuba.popularmovies.data;

import java.util.ArrayList;


public class MovieListResponse {
    private int page;
    private ArrayList<Movie> results=new ArrayList<>();
    private int total_results;
    private int total_pages;

    public ArrayList<Movie>  getResults() {
        return results;
    }

    public int getTotal_pages() {
        return total_pages;
    }
}

File: app/src/main/java/com/koczuba/popularmovies/data/MovieListResponse.kt
package com.koczuba.popularmovies.data

import java.util.ArrayList

/**
 * https://developers.themoviedb.org/3/movies/get-popular-movies
 * https://developers.themoviedb.org/3/movies/get-top-rated-movies
 */

class MovieListResponse(val page: Int,
                        val results: ArrayList<Movie> = ArrayList<Movie>(),
                        val total_results: Int,
                        val total_pages: Int)